tests/cli/fluvio_smoke_tests/producer-smartmodule.bats

```diff
@@ -0,0 +1,361 @@
+#!/usr/bin/env bats
+
+TEST_HELPER_DIR="$BATS_TEST_DIRNAME/../test_helper"
+export TEST_HELPER_DIR
+
+load "$TEST_HELPER_DIR"/tools_check.bash
+load "$TEST_HELPER_DIR"/fluvio_dev.bash
+load "$TEST_HELPER_DIR"/bats-support/load.bash
+load "$TEST_HELPER_DIR"/bats-assert/load.bash
+
+# disabling compiling because it has no feedback
+# this should be done before running test
+setup_file() {
+    # Compile the smartmodule examples
+#    pushd "$BATS_TEST_DIRNAME/../../.." && make build_smartmodules && popd
+    SMARTMODULE_BUILD_DIR="$BATS_TEST_DIRNAME/../../../smartmodule/examples/target/wasm32-unknown-unknown/release"
+    export SMARTMODULE_BUILD_DIR
+
+}
+
+@test "invoke map smartmodule in producer by path" {
```

Comment:
In our CI, we run these tests using `stable` and `latest` build for SPU and CLI. I think that we should skip this test for now while we are on `stable` for either CLI and `SPU`.

In PR #3092, this was done to fix that. https://github.com/infinyon/fluvio/pull/3092/commits/35bef9fe30dff63b2d68d03c50995bc4ae25f0a9 

Maybe we can apply something similar for this here. (But we would need to check that either CLI and SPU is on `latest` to run the test.

This will be temporal until we release a new stable version with support to this

Reply:
I see. I added another commit which I think will do this.

Reply:
Perfect!